tests/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ if (OE_SGX)
   set(DEFINE_OE_SGX "-DOE_SGX")
 endif ()
 
+add_subdirectory(invalid_image)
```

Comment:
It turns out the the test is unix-specific. You would need to move the test as part of https://github.com/openenclave/openenclave/blob/dbd3e7ad0c0e4b374b3c63e860ca2698d9984d52/tests/CMakeLists.txt#L151

Reply:
AArch64 fails, so I guess it's SGX-specific? I moved the test to OE_SGX and clarified the comment in the test. Please try again.